scikit/notebooks/Scikit Learn on CMLE.ipynb

```diff
@@ -0,0 +1,381 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part 0: Setup\n",
+    "* [Create a project on GCP](https://cloud.google.com/resource-manager/docs/creating-managing-projects)\n",
+    "* [Create a Google Cloud Storage Bucket](https://cloud.google.com/storage/docs/quickstart-console)\n",
+    "* [Enable Cloud Machine Learning Engine and Compute Engine APIs](https://console.cloud.google.com/flows/enableapi?apiid=ml.googleapis.com,compute_component&_ga=2.217405014.1312742076.1516128282-1417583630.1516128282)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download the data\n",
+    "The [Census Income Data Set](https://archive.ics.uci.edu/ml/datasets/Census+Income) that this sample\n",
+    "uses for training is hosted by the [UC Irvine Machine Learning\n",
+    "Repository](https://archive.ics.uci.edu/ml/datasets/). We have hosted the data\n",
+    "on Google Cloud Storage in a slightly cleaned form:\n",
+    "\n",
+    " * Training file is `adult.data.csv`\n",
+    " * Evaluation file is `adult.test.csv`\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Disclaimer\n",
+    "The source of this dataset is from a third party. Google provides no representation,\n",
+    "warranty, or other guarantees about the validity or any other aspects of this dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Create a directory to hold the data\n",
+    "! mkdir census_data\n",
+    "\n",
+    "# Download the data\n",
+    "! curl https://archive.ics.uci.edu/ml/machine-learning-databases/adult/adult.data --output census_data/adult.data\n",
+    "! curl https://archive.ics.uci.edu/ml/machine-learning-databases/adult/adult.test --output census_data/adult.test"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part 1: Train the model \n",
+    "First the data is loaded into a numpy array that can be used by Scikit Learn. Then a simple model is created and fit against the training data. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import pickle\n",
+    "from sklearn import svm\n",
+    "from sklearn.ensemble import RandomForestClassifier\n",
+    "from sklearn.externals import joblib\n",
+    "import tensorflow as tf  # Tensorflow is only used to retrieve data from the census data files\n",
```

Comment:
import tensorflow is an overkill. Use local file read

Reply:
Removed:

If we switch from using the UCI version vs the one we have in the census sample. Instead of downloading the data, TF.gfile can just access the data directly from the GCS bucket, would that work or would you prefer they still download it?